plugin.py

```diff
@@ -90,6 +94,32 @@ def install_or_update(cls) -> None:
             raise
 
 
+class RustAnalyzerOpenDocsCommand(LspTextCommand):
+    session_name = "rust-analyzer"
+
+    def is_enabled(self) -> bool:
+        if not bool(self.session_by_name(self.session_name)):
+            return False
+
+        selection = self.view.sel()
+        if len(selection) == 0:
+            return False
+
+        return True
```

Comment:
would suggest to call the default implementation and add that selection check only, to avoid repeating the logic.

The original implementation does the session check already.
https://github.com/sublimelsp/LSP/blob/34e9fbe52460e68aba7495cf33c33528f69f63a6/plugin/core/registry.py#L54-L69